.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Add new sources of input to the chat model # Name of the workflow
```

Comment:
You should have this action only for release branch. And this should better be a step after the release build Github action. You can check how to make this depend on that.

Reply:
Also let's name this script as build-chatbot?

Reply:
Done